streams/src/test/java/org/apache/kafka/streams/kstream/internals/GlobalKTableJoinsTest.java

```diff
@@ -64,7 +66,14 @@ public void apply(final String key, final String value) {
                 results.put(key, value);
             }
         };
+    }
 
+    @After
+    public void cleanup() {
+        if (driver != null) {
+            driver.close();
+        }
+        driver = null;
```

Comment:
This is to prevent some test cases that do not actually create a new `driver` instance: in some test classes we have only a few test case that create a new `driver`.